docs/quick_start/index.md

```diff
@@ -82,37 +60,84 @@ backend
 │   ├── ProjectName.IntegrationTests - Comprehensive application-wide tests.
 ```
 
-### Generating initial migrations
+## Setting up the cluster
 
-Let's start with generating initial migrations. Begin by navigating, to the `backend/src/Apps/ProjectName.Migrations` directory. Generate initial migrations using the `dotnet ef` tool (install it via `dotnet tool install --global dotnet-ef`):
+In our development process, we emphasize creating an environment that mirrors production as closely as possible. This approach ensures consistency and reliability across different stages of development, testing, and deployment. To achieve this, we utilize a local Kubernetes cluster, which allows us to simulate the behavior of our applications in a controlled environment that resembles our production setup. Our local Kubernetes cluster, configured in the `dev-cluster` directory, comprises several key services:
+
+- **PostgreSQL Database**: A relational database system for storing and managing application data.
+- **Blob Storage**: A service for storing large amounts of unstructured data, like images or videos.
+- **RabbitMQ**: A message broker that enables applications to communicate with each other asynchronously.
+- **Kratos**: An identity management system that handles user accounts, authentication, and authorization.
+- **Seq**: Log system designed for searching, analysis, and visualization of structured log data.
+- **Jaeger**: An open-source tool for tracing requests.
+- **Traefik**: A cloud-native application proxy that routes incoming requests to the appropriate microservices.
+- **OpenTelemetry**: A set of APIs, SDKs, and tools to generate, collect, and export telemetry data.
+
+To configure the local cluster, start by navigating to the `dev-cluster` directory. Once there, you can optionally set the `sendgrid_api_key` in the `terraform.tfvars` file:
+
+```terraform
+sendgrid_api_key = "my_sendgrid_api_key"
+```
+
+Follow these commands to establish your local cluster, or alternatively, use the `deploy.sh` script for an automated setup:
+
+```sh
+# Remove existing cluster and its registry to ensure a clean state.
+k3d cluster delete projectname
+k3d registry delete k3d-projectname-registry.local.lncd.pl
+rm *.tfstate*
+
+# Log in to Azure Container Registry and pull the Traefik proxy image.
```

Comment:
And, I think, we should also remove this section or at least add a comment that this requires access to LeanCode's ACR and will be fixed in future version.s